example/spec/parallel/testAPI/sqsRuleSpec.js

```diff
@@ -92,6 +93,9 @@ async function cleanUp() {
 
   // TODO there may be another execution to delete here
```

Comment:
Is this still a valid TODO?

Reply:
I believe so. I've made a note about it in the next ticket